docs/getting-started/installing-service-binding.md

```diff
@@ -4,13 +4,16 @@ sidebar_position: 1
 
 # Installing Service Binding
```

Comment:
* Add the Kubernetes and OpenShift compatibility matrix.
* Installation through OpenShift Developer Console (I will add a separate ticket)

Reply:
Done